keps/sig-auth/541-external-credential-providers/README.md

```diff
@@ -129,7 +129,7 @@ users:
   user:
     exec:
       # API version to use when decoding the ExecCredentials resource. Required.
-      apiVersion: "client.authentication.k8s.io/v1beta1"
+      apiVersion: "client.authentication.k8s.io/v1"
```

Comment:
Use `interactiveMode: Never` below to match the docs.

Reply:
a83d7fc9